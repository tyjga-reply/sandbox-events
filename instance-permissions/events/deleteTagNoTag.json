{
    "version": "0",
    "id": "480bee96-b1f2-bec6-6ae5-ae8f277c0769",
    "detail-type": "AWS API Call via CloudTrail",
    "source": "aws.ec2",
    "account": "211125361846",
    "time": "2024-12-17T11:13:59Z",
    "region": "us-east-1",
    "resources": [],
    "detail": {
        "eventVersion": "1.10",
        "userIdentity": {
            "type": "AssumedRole",
            "principalId": "AROATCKANLS3NIDXYUGTV:t.kapp@reply.de",
            "arn": "arn:aws:sts::211125361846:assumed-role/AWSReservedSSO_AdministratorAccess_d519e2a9ed8243a4/t.kapp@reply.de",
            "accountId": "211125361846",
            "accessKeyId": "ASIATCKANLS3JW2WSF6Z",
            "sessionContext": {
                "sessionIssuer": {
                    "type": "Role",
                    "principalId": "AROATCKANLS3NIDXYUGTV",
                    "arn": "arn:aws:iam::211125361846:role/aws-reserved/sso.amazonaws.com/eu-central-1/AWSReservedSSO_AdministratorAccess_d519e2a9ed8243a4",
                    "accountId": "211125361846",
                    "userName": "AWSReservedSSO_AdministratorAccess_d519e2a9ed8243a4"
                },
                "attributes": {
                    "creationDate": "2024-12-17T08:29:50Z",
                    "mfaAuthenticated": "false"
                }
            }
        },
        "eventTime": "2024-12-17T11:13:59Z",
        "eventSource": "ec2.amazonaws.com",
        "eventName": "DeleteTags",
        "awsRegion": "us-east-1",
        "sourceIPAddress": "91.102.9.132",
        "userAgent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/18.1.1 Safari/605.1.15",
        "requestParameters": {
            "resourcesSet": {
                "items": [
                    {
                        "resourceId": "i-003492c2f4628ec51"
                    }
                ]
            },
            "tagSet": {
                "items": [
                    {
                        "key": "permission",
                        "value": "yes"
                    }
                ]
            }
        },
        "responseElements": {
            "requestId": "e685fa87-4595-478c-9fa6-1ad6298cc50a",
            "_return": true
        },
        "requestID": "e685fa87-4595-478c-9fa6-1ad6298cc50a",
        "eventID": "98809855-2ac6-4b2a-8d5c-88c2c52db67a",
        "readOnly": false,
        "eventType": "AwsApiCall",
        "managementEvent": true,
        "recipientAccountId": "211125361846",
        "eventCategory": "Management",
        "tlsDetails": {
            "tlsVersion": "TLSv1.3",
            "cipherSuite": "TLS_AES_128_GCM_SHA256",
            "clientProvidedHostHeader": "ec2.us-east-1.amazonaws.com"
        },
        "sessionCredentialFromConsole": "true"
    }
}