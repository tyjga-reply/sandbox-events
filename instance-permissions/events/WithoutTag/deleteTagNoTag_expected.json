{
    "statusCode": 200,
    "body": "{\"message\": \"Resource ID extracted successfully!\", \"eventName\": \"DeleteTags\", \"resourceId\": \"i-003492c2f4628ec51\", \"valid\": false, \"ipInfo\": {\"PrivateIpAddress\": \"172.31.25.223\"}}"
  }